src/libcollections/btree/map.rs

```diff
@@ -764,7 +764,7 @@ impl<K: Ord, V> BTreeMap<K, V> {
     /// let mut map: BTreeMap<&str, i32> = ["Alice", "Bob", "Carol", "Cheryl"].iter()
     ///                                                                       .map(|&s| (s, 0))
     ///                                                                       .collect();
-    /// for (_, balance) in map.range_mut(Included("B"), Excluded("Cheryl")) {
+    /// for (_, balance) in map.range_mut::<str, _>((Included("B"), Excluded("Cheryl"))) {
```

Comment:
Seems a bit weird that this would be required now?

The argument can be shortened to `"B".."Cheryl"`.